packages/@react-native-windows/cli/package.json

```diff
@@ -23,7 +23,7 @@
     "glob": "^7.1.1",
     "inquirer": "^3.0.6",
     "mustache": "^4.0.1",
-    "nuget-exe": "4.9.2",
+    "nuget-exe": "5.8.0-0",
```

Comment:
If we want to use the version from master, these need to use the non "-0" version. We can actually just make that change, instead of the rest.

Reply:
See https://www.npmjs.com/package/nuget-exe

Newer versions of beachball will earn you of you have a public package depending on a  private one, but I'm not sure if the version in this branch will.